engine/apps/mobile_app/demo_push.py

```diff
@@ -0,0 +1,79 @@
+import json
+
+from fcm_django.models import FCMDevice
+from firebase_admin.messaging import APNSPayload, Aps, ApsAlert, CriticalSound, Message
+
+from apps.mobile_app.tasks import FCMMessageData, MessageType, _construct_fcm_message, _send_push_notification
+from apps.user_management.models import User
+
+
+def send_test_push(user, critical=False):
+    device_to_notify = FCMDevice.objects.filter(user=user).first()
+    message = _get_test_escalation_fcm_message(user, device_to_notify, critical)
+    _send_push_notification(device_to_notify, message)
+
+
+def _get_test_escalation_fcm_message(user: User, device_to_notify: FCMDevice, critical: bool) -> Message:
+    # TODO: this method is copied from _get_alert_group_escalation_fcm_message
+    # to have same notification/sound/overrideDND logic. Ideally this logic should be abstracted, not repeated.
+    from apps.mobile_app.models import MobileAppUserSettings
+
+    thread_id = f"test_push"
```

Comment:
Maybe add some randomness (e.g. random uuid) to make sure notifications are not grouped?